### PR TITLE
Add function to get sys services

### PR DESCRIPTION
**Pull request by anjalshireesh**
The intention is to provide status of any sys services that can
potentially impact the performance of minio.

At present, it will return information about the `selinux` service
(not-installed/disabled/permissive/enforcing)

### CODE REVIEW

File: health.go [hunk 1]
```diff
@@ -267,7 +291,58 @@ func GetOSInfo(ctx context.Context, addr string) OSInfo {
...
+			// selinux is not installed.
+			ss.Status = SrvNotInstalled
+			return ss, nil
+		} else {
+			return ss, err
+		}
+	}
+
+	file, err := os.Open(configFile)
+	if err != nil {
```
Comment (balamurugana) [inline health.go:325]: You could avoid `os.Stat()` and do the check like
```go
if errors.Is(err, os.ErrNotExist) {
    ss.Status = SrvNotInstalled
    return ss, nil
}
```

File: health.go [hunk 2]
```diff
@@ -267,7 +291,58 @@ func GetOSInfo(ctx context.Context, addr string) OSInfo {
...
+		return ss, err
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	statuses := []string{"enforcing", "permissive", "disabled"}
+	for scanner.Scan() {
+		txt := scanner.Text()
+		for _, status := range statuses {
+			if strings.Contains(txt, "SELINUX="+status) {
```
Comment (balamurugana) [inline health.go:335]: you could simplify this by
```go
tokens := strings.SplitN(strings.TrimSpace(scanner.Text()), "=", 2)
if len(tokens) == 2 && tokens[0] == "SELINUX" {
    ss.Status = tokens[1]
    return ss, nil
}
```

File: health.go (hunk 1 above)
Comment (anjalshireesh) [reply health.go]: Done

File: health.go (hunk 2 above)
Comment (anjalshireesh) [reply health.go]: Done

**Review by balamurugana — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**